src/MoBi.Presentation/Presenter/SelectFolderAndIndividualFromProjectPresenter.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using MoBi.Core.Domain.Repository;
+using MoBi.Presentation.DTO;
+using MoBi.Presentation.Presenter.BasePresenter;
+using MoBi.Presentation.Tasks.Edit;
+using MoBi.Presentation.Views;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Presentation.Presenters;
+
+namespace MoBi.Presentation.Presenter
+{
+   public interface ISelectFolderAndIndividualFromProjectPresenter : IDisposablePresenter
+   {
+      /// <summary>
+      ///    Opens a dialog where the user will select an IndividualBuildingBlock from the project and a file path to export
+      /// </summary>
+      (string, IndividualBuildingBlock) GetPathAndIndividualForExport(IContainer container);
```

Comment:
I think you can name the return values also but I am not sure (string exportPath, xxx)